dashboard/tiles/latency/main.tf

```diff
@@ -0,0 +1,39 @@
+variable "title" { type = string }
+variable "group_by_fields" { default = [] }
+variable "filter" { type = list(string) }
```

Comment:
nit: split `variables.tf` and `outputs.tf` 🙏

Reply:
oh fun, the whole thing is inputs/outputs 🤔

Reply:
Yeah, these tiles are really simple little things. I could split them up but it's just more tab-flipping when you iterate on them.